README.md

```diff
@@ -169,7 +169,7 @@ userRouter.init(
   });
 ```
 
-Note: Setting the `authResponseExclusionList` array as `['password', 'banner']` will prevent these fields from appearing in the authentication response. By default, the `password` field is removed from the response. To allow all fields to be sent, set `authResponseExclusionList` to an empty array.
+Note: Setting the `userExclusionList` array as `['password', 'banner']` will prevent these fields from appearing in the authentication response. By default, the `password` field is removed from the response. To allow all fields to be sent, set `userExclusionList` to an empty array.
```

Comment:
Ah yes, good spot. I changed this naming a few times to make it clearer for an end user. Thanks, wil fix now.